.setup/SUBMITTY_TEST.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+current_dir=$(pwd)
+
+cd /usr/local/submitty/GIT_CHECKOUT/Submitty/site || {
```

Comment:
Consider using `pushd`/`popd` instead.  See [here](https://github.com/Submitty/Lichen/blob/7e7d22975cb9b4c62a26fa8255002969974bafb4/install_lichen.sh#L57) for an example.

Reply:
Great point, Thanks